ui/raidboss/data/03-hw/raid/a5s.js

```diff
@@ -289,6 +289,7 @@ let bombLocation = (matches) => {
         '(?<!Hummel)Faust': 'Faust',
         '(?<!Smart)Bomb': 'Bombe',
         'Hummelfaust': 'Hummelfaust',
+        'Gobbledygroper': 'Gobgreifer',
```

Comment:
Sorry, it is hard enough to review removals from the timeline section on its own, but it becomes much harder when you are also making additions and modifications.

If you'd like to clean up unused entries, can you make this patch *just* removals?

Reply:
reverted all additions (except the needed missing translation flag for one of the files)
Once this PR is done, the addiotions will be made again